app/controllers/staff/activities_controller.rb

```diff
@@ -50,7 +50,7 @@ def show
       format.xml do |_format|
```

Comment:
Not about this LOC, but there's a typo in the commit message " Convert translations and switch controlle to 'Actual' "

Reply:
fixed, thks